app/helpers/makePlutoToastAction.ts

```diff
@@ -1,7 +1,10 @@
+import PlutoRenderer from "..";
 import { INotificationActionPayload } from "./notifier";
+import { ACTION_TYPES } from "../actions/actionTypes";
 
 export default function alertToast(notificationActionPayload: INotificationActionPayload): void {
-  if (!!notificationActionPayload) {
```

Comment:
is it safe?

Reply:
Yes, because that payload is not optional property.